source/Beacon-GS64-Tests/SignalLoggerTest.class.st

```diff
@@ -0,0 +1,21 @@
+Class {
+	#name : #SignalLoggerTest,
+	#superclass : #TestCase,
+	#category : #'Beacon-GS64-Tests'
+}
+
+{ #category : #tests }
+SignalLoggerTest >> testStartingTwiceDoesNotRegisterTwice [
+	| initialNumberOfSubscriptions |
+	initialNumberOfSubscriptions := Beacon instance announcer numberOfSubscriptions.
+	SignalLogger resetInstance.
+	SignalLogger start.
+	SignalLogger start.
```

Comment:
Is it on purpose to send `start` twice?

Reply:
Yes the test case checks starting twice does not generate more than one subscription